packages/server/tests/acceptance/estimateGasPrecompile.spec.ts

```diff
@@ -404,14 +597,52 @@ describe('EstimatePrecompileContract tests', function () {
     isWithinDeviation(gasResult.gasUsed, estimateGasResponse, lowerPercentBound, upperPercentBound);
   });
 
+  //EGP-015
+  it('should call estimateGas with approveNFT function', async function () {
+    let spender = prefix + EstimatePrecompileContractAddress;
+    const nftSerial = await mintNFT();
+
+    //approve EstimatePrecompileContract to use the NFT
+    // const nftTokenContract = new ethers.Contract(nftAddress, ERC721MockJson.abi, accounts[0].wallet);
+    const nftApproveTX = await nftTokenContract.approve(spender, nftSerial, { gasLimit: 1_000_000 });
```

Comment:
Can you separate logically the different steps with empty lines for better readability.

Reply:
done